config/express.js

```diff
@@ -24,15 +24,15 @@ module.exports = function(app, passport, mongoose) {
 
     //Don't use logger for test env
     if (process.env.NODE_ENV !== 'test') {
-        app.use(express.logger('dev'));
+        app.use(express.logger('dev');
```

Comment:
Ok i just saw it, i'll fix immediately. But should i be too concerned about the houndci notifications